site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/FileTable.js

```diff
@@ -0,0 +1,95 @@
+import _get from "lodash/get";
+import PropTypes from "prop-types";
+import React from "react";
+import {
+    Icon,
+    Popup,
+    Table,
+} from "semantic-ui-react";
+import { humanReadableBytes } from "react-invenio-deposit";
+
+
+const FileTableRow = ({ handleDelete, file, decimalSizeDisplay }) => {
+    return (
+        <Table.Row key={file.name}>
+            <Table.Cell data-label={"Filename"} width={10}>
+                <div>
```

Comment:
```suggestion
                <>
```
can be replaced by react fragment, no div needed